templates/kafka/0/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+zk:
```

Comment:
Zookeeper it's not a copy, neither it's not based in yours, i don't understand why you say it??. I build my own and i deploy it as you can see in docker-compose file. 
Reasons to kafka have it's own zookeeper:
First reason is that mine is about 130 Mb smaller than yours. :P
This is yours - rancher/zookeeper      3.4.6-2    312 MB
This is mine - rawmind/rancher-zk    3.4.6-1   182.4 MB

Second reason is that mine could scale and get reconfigured dinamically, with confd obtaining data from rancher-metadata v2. 

Third reason is that in mine you could configure memory sizing when you launch it.

Fouth reason is that kafka recommends have a dedicated zookeeper to work well in terms of performance.